esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -400,8 +398,8 @@ impl<'d> Rtc<'d> {
         h.write(|w| unsafe { w.bits((boot_time_us >> 32) as u32) });
     }
 
-    /// Get the current time.
-    pub fn current_time(&self) -> NaiveDateTime {
+    /// Get the current time in microseconds.
+    pub fn current_time_us(&self) -> u64 {
```

Comment:
Let's keep the API symmetrical. I think we should drop the `_us` postfix here, the documentation is enough.

Reply:
I'd rather be explicit, someone will have their autocomplete write `jiff::Timestamp::from_seconds(rtc.current_time())`, and that should be an obvious mistake.

Reply:
Fair enough, as long as they are consistent I'm happy.